src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -359,6 +370,21 @@ void P093_data_struct::applySettings() {
     packet[7] |= 0x01;
   }
 
+  if (_writeStatus.isDirty(RemoteTemperature)) {
+    packet[5] |= 0x07;
+    if(_wantedSettings.remoteTemperature > 0) {
+      if (_tempMode) {
+        packet[8] = static_cast<uint8_t>(_wantedSettings.remoteTemperature * 2.0f + 128.0f);
```

Comment:
This doesn't fit in `uint8_t`.
Only if the remoteTemperature is -127 ... 0, it will fit in `uint8_t`
And this will not happen as the temperature > 0 as checked before.

Reply:
But why it is used the same way in line 341 to set temperature?

Reply:
Ah I need some more coffee....
My bad.
It allows to set temperatures from -63 ... 63

Sorry, I will now get my coffee :)